smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/StructureGenerator.kt

```diff
@@ -128,7 +129,11 @@ class StructureGenerator(
             var (memberName, memberSymbol) = memberShapeDataContainer.getOrElse(it) { return@forEach }
             writer.writeMemberDocs(model, it)
             val indirect = it.hasTrait<SwiftBoxTrait>()
-            val indirectOrNot = "@Indirect ".takeIf { indirect } ?: ""
+            var indirectOrNot = ""
+            if (indirect) {
+                writer.addImport(ClientRuntimeTypes.Core.Indirect)
+                indirectOrNot = "@Indirect "
```

Comment:
`ClientRuntimeTypes.Core.Indirect` is imported directly when the `@Indirect` property wrapper is referenced